website/content/2025-09-29-introducing-hypofuzz.md

```diff
@@ -0,0 +1,28 @@
+---
+date: 2025-08-07 00:00
+title: Introducing HypoFuzz
+author: Liam & Zac
+---
+
+**TL;DR:** [HypoFuzz](https://hypofuzz.com/) is a fuzzing backend and dashboard for your Hypothesis tests. It's designed for deep bug discovery, observability, and autoscaling, and effortlessly combines property-based testing and fuzzing. [Try it today](https://hypofuzz.com/docs/quickstart.html); we'd love to hear your feedback.
```

Comment:
It strikes me that there's no easy-to-follow link from the quickstart page (or docs in general) back to the main website, or specifically the pricing page.  We might want to add that on the Hypofuzz side.